backend/src/chats/chat-user-permissions/entities/chat-user-permission.entity.ts

```diff
@@ -43,4 +45,10 @@ export class ChatUserPermission {
 	})
 	@JoinColumn({ name: 'permission_id' })
 	permissions: Permission[];
+
+	@CreateDateColumn()
+	created_at: Date;
+
+	@UpdateDateColumn()
+	updated_at: Date;
 }
```

Comment:
not sure why you need time and date for the permissions? but you prob have a good reason :)

Reply:
ehhhhhhhhh yes.. there was something.. I kinda forgot..
but it doesn't hurt to have it, and it's auto generated/auto updated etc, so it should not be in the way of anything even if I never remember what it was for..

Reply:
haha alright